examples/express-app/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const MoltinGateway = require('@moltin/sdk').gateway
+
+const app = express()
+
+const Moltin = MoltinGateway({
+  client_id: 'j6hSilXRQfxKohTndUuVrErLcSJWP15P347L6Im0M4'
```

Comment:
@notrab ENV var?

Reply:
I think it's overly complex for this example. The `client_id` provided is I Love Lamp which we should assume is always production-ready.

I'd be happy to update this line to be `process.env.MOLTIN_CLIENT_ID || 'j6hSilXRQfxKohTndUuVrErLcSJWP15P347L6Im0M4'` but you'd end up having to add `dotenv` etc to make this work properly. Just feels complex for a basic example - that's to be used during development.

Thoughts?